### PR TITLE
first open source contribution

### PR DESCRIPTION
**Pull request by mohdUsman-6508**

### CODE REVIEW

**Comment by mohdUsman-6508** [comment]
It was fun.

File: include.js [hunk 1]
```diff
@@ -1261,6 +1261,14 @@ let cards = [
     author: 'Divyansh',
     githubLink: 'https://github.com/okaydivyansh'
   },
+
+  {
+    artName: 'wheel', // change this to the name of your artwork
+    pageLink: './Art/mohdUsman-6508/triangle.html', // change this
+    imageLink: './Art/mohdUsman-6508/wheel.gif', // change this
+    author: 'Mohd Usman', // use your name
+    githubLink: 'https://github.com/mohdUsman-6508' // change this
```
Comment (sophiabrandt) [inline include.js:1270]: Can you please remove all the comments from your entries? Thank you.

File: include.js [hunk 2]
```diff
@@ -1261,6 +1261,14 @@ let cards = [
     author: 'Divyansh',
     githubLink: 'https://github.com/okaydivyansh'
   },
+
+  {
+    artName: 'wheel', // change this to the name of your artwork
+    pageLink: './Art/mohdUsman-6508/triangle.html', // change this
```
Comment (sophiabrandt) [inline include.js:1267]: The file name is incorrect, it needs to be named after the file name of _your_ contribution, so it's `wheel.html`.

File: Art/mohdUsman-6508/wheel.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Animated Cycle Wheel</title>
+    <link rel="stylesheet" href="wheel.css" />
```
Comment (sophiabrandt) [inline Art/mohdUsman-6508/wheel.html:7]: Your `wheel.css` file is empty, so I am not sure how you plan to animate your contribution. Without checking out your code, I am assuming that you only have a few empty `div`s.

Did you forget to add your changes to `wheel.css`?

**Review by sophiabrandt — CHANGES_REQUESTED** [review]

File: Art/mohdUsman-6508/wheel.html (hunk 3 above)
Comment (mohdUsman-6508) [reply Art/mohdUsman-6508/wheel.html:7]: yes, i forgot

File: include.js (hunk 2 above)
Comment (mohdUsman-6508) [reply include.js:1274]: i have chaged it.

File: include.js (hunk 1 above)
Comment (mohdUsman-6508) [reply include.js:1277]: i have removed all the comments.

**Comment by mohdUsman-6508** [comment]
Thank you very much @sophiabrandt  for being with me to all the way to end. 
 Best Wishes.

**Pull request merged**